book/survival.qmd

```diff
@@ -15,100 +15,215 @@ TODO
+Often, the different types of right-censorig can co-occur in any given data set.
+
+For the purposes of this book, however, the difference will be largely irrelevant, as the different types are treated similarly at the modeling/estimation stage.
+
+
+#### Left-censoring {.unnumbered .unlisted}
+Left-censoring is a rarer form of censoring and occurs when the event happens at some unknown time before the study start.
+While quiet rare in medical settings, this type of data often occurs in sociology studies when retrospective interviews are conducted.
+
+For illustration, consider an interview whith questions alas "At what age did you ... for the first/last time?", where you can fill in the blank with "use a smartphone", "drink alcohol" or similar.
+Let $A_i$ denote the age of the participants during the interview.
+If they remember the age exactly\footnote{If one assumes that the subjects remember correctly, otherwise one would need to account for potential measuresment error}, then $T_i = Y_i \leq A_i$.
+If the event didn't occur yet, then the subject is right-censored at $A_i$, i.e., $Y_i > T_i = C_i = A_i$.
+However, if the event occured, but the subjects don't remember when, the subject is left-censored at $A_i$, i.e. we know that $T_i = Y_i < A_i$, but not the exact time.
+
+Note that for this type of data you need an additional variable in order to differentiate between left- and right-censoring as well as exact event times.
+In practice this is often solved by including two columns that store time, such that left-censored data is given by $(-\infty, A_i)$, while right-censored data is denoted by $(A_i, \infty)$.
+In this convention, actual event times are denoted by $(T_i, T_i)$.
+
 
-Censoring is often defined as *uninformative* if $Y \indep C$ and *informative* otherwise however these definitions can be misleading as the term 'uninformative' appears to be imply that censoring is independent of both $X$ and $Y$, and not just $Y$. Instead the following more precise definitions are used in this report.
+#### Interval-censoring {.unnumbered .unlisted}
+
+Interval-censoring occurs when the event takes place in some interval within the study period, but the exact time of event is unknown.
+This often occurs in data resulting from regular or irregular check-ups, as it often occurs in electronic health record data.
+For example, patients might get periodic tests for skin cancer.
+Some patients might check once a year, for others the intervals between checks might be longer.
+If skin cancer is detected, one only knows that it occured between the last and current check up, but not the exact time.
+
+Formally, consider $L_i$ the subject-specific check-up time and $R_i$ the current check-up time.
+Then we know that if an event occured $T_i = Y_i \in (L_i, R_i]$.
+If no event occured then $Y_i > T_i = C_i = R_i$.
```

Comment:
I've simplified this, but don't think we need repeated measures. While check ups occur regularly in different intervals. We always only need the last interval, because then the event either occurred ($Y_i \in (L_i, R_i]$) or didn't, so $R_i = C_i$. We don't need the info about previous intervals for interval censoring definition.